Presentation.Web/Controllers/API/ItContractRightsController.cs

```diff
@@ -26,6 +29,17 @@ public HttpResponseMessage GetRightsForUser(int userId)
                 var theRights = new List<ItContractRight>();
                 theRights.AddRange(RightRepository.Get(r => r.UserId == userId));
 
+                //Get signed contracts
+                var contracts = _objectRepository.Get(c => c.ContractSignerId == userId);
+
+                foreach (var contract in contracts ?? Enumerable.Empty<ItContract>())
+                {
+                    var signerRole = new ItContractRole() { Name = "Kontraktunderskriver", HasReadAccess = true, HasWriteAccess = false };
+                    var signerRight = new ItContractRight() { ObjectId = contract.Id, Object = contract, Role = signerRole };
+
+                    theRights.Add(signerRight);
```

Comment:
Would prefer if it was added to the DTOs as adding it to the entities could be misleading. Someone might call save changes and cause all kinds of problems.